source/plugins/go/src/telemetry.go

```diff
@@ -262,11 +267,16 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 				SendEvent(eventNameDaemonSetHeartbeat, telemetryDimensions)
 				flushRateMetric := appinsights.NewMetricTelemetry(metricNameAvgFlushRate, flushRate)
 				TelemetryClient.Track(flushRateMetric)
+
 				logRateMetric := appinsights.NewMetricTelemetry(metricNameAvgLogGenerationRate, logRate)
 				logSizeMetric := appinsights.NewMetricTelemetry(metricNameLogSize, logSizeRate)
 				TelemetryClient.Track(logRateMetric)
 				Log("Log Size Rate: %f\n", logSizeRate)
 				TelemetryClient.Track(logSizeMetric)
+
+				metadataSizeMetric := appinsights.NewMetricTelemetry(metricNameMetadataSize, metadataSizeRate)
+				TelemetryClient.Track(metadataSizeMetric)
```

Comment:
we should emit this metric only when KubernetesMetadataEnabled .

if KubernetesMetadataEnabled  {
    metadataSizeMetric := appinsights.NewMetricTelemetry(metricNameMetadataSize, metadataSizeRate)
				TelemetryClient.Track(metadataSizeMetric)
}